src/components/ebay-success-message/index.marko

```diff
@@ -0,0 +1,25 @@
+<script marko-init>
+    var processHtmlAttributes = require("../../common/html-attributes");
+</script>
+
+<section
+    aria-labelledby=widget.elId("title")
+    class=["success-message", data.full && "success-message--full", data.class]
+    style=data.style
+    role="region"
+    ${processHtmlAttributes(data)}>
+
+    <${data.titleHeadingTag || "h2"} id=widget.elId("title") class="success-message__title">
+        <span class="success-message__status" role="img" aria-label=data.iconA11yText></span>
+        <span>${data.title}</span>
+    </>
+
+    <p
+        class=["success-message__content", data.content.class]
+        style=data.content.style
+        if(data.content && data.content.renderBody)>
+        <invoke data.content.renderBody(out) />
+    </>
```

Comment:
I know this is an easy convention, but we usually close non-dynamic tags so there's no confusion.
```suggestion
    </p>
```